tardis/model/base.py

```diff
@@ -73,6 +73,15 @@ def __init__(self, velocity, homologous_density, abundance, time_explosion,
         self.homologous_density = homologous_density
         self._abundance = abundance
         self.time_explosion = time_explosion
+
+        self.raw_abundance = self._abundance
+        if isotope_abundance is not None:
+            self.raw_isotope_abundance = isotope_abundance
+            self._abundance = self.raw_isotope_abundance.as_atomic_numbers(
```

Comment:
@yeganer If I put this line in under `abundance` property, then this line would be executed _every time_ `model.abundance` is called , and I think it should only be called 1 time during initialization.